src/app/common/feature-vtabs.component.ts

```diff
@@ -57,7 +55,7 @@ export class GlobalModuleReference {
     // NOTE: when [routerLink] is used, Angular automatically set tabindex="0". In order to avoid that behavior, we decided to add tabindex="-1".
     template: `
 <div class="sidebar crumb" [class.nav]="IsActive">
```

Comment:
please rename this to isActive from IsActive, too.

Reply:
update